sdk/resource/resource_test.go

```diff
@@ -769,3 +770,30 @@ func TestWithContainer(t *testing.T) {
 		string(semconv.ContainerIDKey): fakeContainerID,
 	}, toMap(res))
 }
+
+func TestResourceRace(t *testing.T) {
+	// Creating Resources should also be free of any ASAN issues,
```

Comment:
I think that "ASan" is more of a C/C++ term
```suggestion
	// Creating Resources should also be free of any data races,
```